cmd/builder/internal/command.go

```diff
@@ -110,7 +111,7 @@ configuration is provided, ocb will generate a default Collector.
 	if err := cmd.Flags().MarkDeprecated(distributionModuleFlag, "use config distribution::module"); err != nil {
 		return nil, err
 	}
-
+	cmd.Flags().BoolVar(&cfg.Distribution.Debug, distributionDebugFlag, false, "Compile with debug flags (default false)")
```

Comment:
We are moving away from flags in favor of config files only. It might be better not to add a new flag at this point.

Reply:
removed